api/prisma/schema.prisma

```diff
@@ -67,6 +67,16 @@ model UserRepoData {
     @@id([userId, repoId])
 }
 
+model YDocSnapshot {
+    id        String   @id
+    createdAt DateTime @default(now())
+    message   String?
+    yDocBlob  Bytes?
+    deleted   Boolean  @default(false)
```

Comment:
I don't want to add `deleted` here since it adds complexity. We could reply on DB backup if we really want to ensure data safety.

Reply:
I am not so familiar with DB backup, any reference for its pros and cons? I am considering a scenario where users might decide to delete a snapshot, but then want to undo the deletion later. Furthermore, if there are 1,000 users performing these operations at different times, can DB backups gracefully address this scenario?

Reply:
I'd skip this scenario for now. If the user deletes it, it is deleted.

Reply:
> I'd skip this scenario for now. If the user deletes it, it is deleted.

OK, let me revert the change.